### DISCUSSION TITLE
Modernize settings view

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#56] [Ideas]
@qw123wh, @odmfl, @Nilsu11:
Could you please test the apk below and give me your opinion?

The changes concern the settings view.
Currently, I find it confusing to have almost all the settings on the same page. So I've separated each category properly.
It will also be easier to add features and maintain settings files.

If you like it (and I hope you do), I'll publish the new version.

[Clock_2.4-debug.zip](https://github.com/user-attachments/files/15796109/Clock_2.4-debug.zip)

**Comment by odmfl** [comment]
Yes I like it, now there is more order in the settings

**Comment by qw123wh** [comment]
Yes, excellent solution

**Comment by BlackyHawky** [comment]
Thank you for your feedback.
I therefore consider that the majority wins. 😅
